sonar-ruby-plugin/src/main/java/org/sonarsource/ruby/converter/RubyVisitor.java

```diff
@@ -114,6 +115,9 @@ public Tree visitNode(AstNode node, List<Object> children) {
     switch (node.type()) {
       case "and":
         return createLogicalOperation(node, children, Operator.CONDITIONAL_AND);
+      case "arg":
+      case "optarg":
```

Comment:
I think `optarg` should wrap the `ParameterTree` around a native tree, otherwise we're loosing information when converting to slang AST (we're losing the parameter default value node)